pkg/manager/member/dm_master_member_manager.go

```diff
@@ -539,7 +542,10 @@ func getNewMasterSetForDMCluster(dc *v1alpha1.DMCluster, cm *corev1.ConfigMap) (
 	dcName := dc.Name
 	baseMasterSpec := dc.BaseMasterSpec()
 	instanceName := dc.GetInstanceName()
-	masterConfigMap := cm.Name
+	masterConfigMap := ""
+	if cm != nil {
```

Comment:
What if cm is nil here, should we return error?

Reply:
I think it's unnecessary🤔